README.md

```diff
@@ -2,6 +2,33 @@
 
 UpSwyng is a mobile-ready, digital directory of resources to assist the unhoused and at-risk communities. This is a [Code For Boulder](https://www.codeforboulder.org) project coming soon to Boulder County, Colorado.
 
+## Before you begin
+
+You can save yourself some grief and aggravation by downloading the versions of the following programs as specified in https://github.com/CodeForBoulder/upswyng/blob/master/package.json
+You will need to check that you have each of the following using the which command or add -v or --version to the name of the program:
+
+0. xcode ==> ``` /usr/bin/xcodebuild -version ```
+   * Install: ``` sudo xcode-select --install ```
+1. ```brew --version``` (called homebrew)
+   * Install: ```/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)" ```
```

Comment:
Instead of having commands for installing these, it's better to link to installation instructions. That way we don't risk these being out of date.

Reply:
K:)